modules/masterbar.php

```diff
@@ -18,3 +20,13 @@
 
 new Masterbar();
 new Admin_Color_Schemes();
+
+/**
+ * Whether to load the admin menu functionality.
+ *
+ * @use add_filter( 'jetpack_load_admin_menu_class', '__return_true' );
+ */
+if ( apply_filters( 'jetpack_load_admin_menu_class', false ) ) {
```

Comment:
Could you add a docblock here, for the codex?

```suggestion
/**
 * Whether to load the admin menu functionality.
 * @use add_filter( 'jetpack_load_admin_menu_class', '__return_true' );
 *
 * @module masterbar
 *
 * @since 9.2.0
 *
 * @param bool $load_admin_menu_class Load Jetpack's custom admin menu functionality. Default to false.
 */
if ( apply_filters( 'jetpack_load_admin_menu_class', false ) ) {
```